junit5/src/main/java/com/linecorp/armeria/testing/junit5/client/HttpDataAssert.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.testing.junit5.client;
+
+import static com.google.common.base.Preconditions.checkState;
+import static java.util.Objects.requireNonNull;
+
+import java.nio.charset.Charset;
+import java.util.Arrays;
+
+import com.linecorp.armeria.common.HttpData;
+
+/**
+ * Assertion methods for HttpData
+ */
+public final class HttpDataAssert extends AssertThat<HttpData, TestHttpResponse> {
+    HttpDataAssert(HttpData actual, TestHttpResponse back) {
+        super(actual, back);
+    }
+
+    public TestHttpResponse isEqualTo(HttpData expected) {
+        requireNonNull(expected, "expected");
+        checkState(actual().equals(expected), "\nexpected: %s\n but was: %s", expected, actual());
```

Comment:
`java.lang.AssertionError` should be raised instead of `IllegalStateException`.
What do you think of delegating https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html since `junit5` is the module name?

Reply:
Thanks. I agree with your opinion. I will replace checkState with assertEquals in JUnit5

Actually, I tried AssertJ api but implementation of junit5 module doesn't depends on assertj so I used `checkState` method. I have a question at this point. junit5 module of armeria already depends on `libs.junit5.jupiter.api` but it doesn't depend on `libs.assertj`. Would it be weired if I use `AssertJ#assertThat` rather than `Junit5#assertEquals`?